Makefile

```
@@ -1,12 +1,15 @@
 MDTOOL ?= /Applications/Visual\ Studio.app/Contents/MacOS/vstool
+NUSPEC_FILE ?= ModernHttpClient.nuspec
 
 .PHONY: all clean
 
 all: ModernHttpClient.iOS64.dll ModernHttpClient.Android.dll ModernHttpClient.Portable.dll
 
+release-pack: version-bump package
+
 package: ModernHttpClient.iOS64.dll ModernHttpClient.Android.dll ModernHttpClient.Portable.dll
 	nuget pack
-	mv modernhttpclient*.nupkg ./build/
+	mv modernhttpclient*.nupkg ~/.local/share/NuGet/Cache/
```

should leave this as ./build/
```suggestion
	mv modernhttpclient*.nupkg ./build/
```

---

Reverted